src/mds/FSMap.cc

```diff
@@ -224,6 +224,51 @@ void FSMap::print(ostream& out) const
   }
 }
 
+void FSMap::print_daemon_summary(ostream& out) const
+{
+  // this appears in the "services:" section of "ceph status"
+  int num_up = 0, num_in = 0, num_failed = 0;
+  for (auto& [fscid, fs] : filesystems) {
+    num_up += fs->mds_map.get_num_up_mds();
+    num_in += fs->mds_map.get_num_in_mds();
+    num_failed += fs->mds_map.get_num_failed_mds();
+  }
+  int num_standby = standby_daemons.size();
+  out << num_up << "/" << num_in << " daemons up";
+  if (num_failed) {
+    out << " (" << num_failed << " failed)";
+  }
+  if (num_standby) {
+    out << ", " << num_standby << " standby";
+  }
+}
+
+void FSMap::print_fs_summary(ostream& out) const
+{
+  // this appears in the "data:" section of "ceph status"
+  if (!filesystems.empty()) {
+    int num_degraded = 0, num_stopped = 0, num_healthy = 0;
+    for (auto& [fscid, fs] : filesystems) {
+      if (fs->mds_map.is_degraded()) {
+	++num_degraded;
+      } else if (fs->mds_map.is_stopped()) {
+	++num_stopped;
+      } else {
+	++num_healthy;
+      }
+    }
+    out << "    volumes: "
```

Comment:
Can we use a different word here? Maybe "cephfs", "fs", or "files"? I'm concerned that using "volumes" will confuse ceph-volume users when we say they have only 1 healthy.